src/yargs-fastly.js

```diff
@@ -12,9 +12,9 @@
 
 module.exports = function commonArgs(yargs) {
   return yargs
-    .option('fastly-namespace', {
-      alias: 'fastlyNamespace',
-      describe: 'CDN Namespace (e.g. Fastly Service ID)',
+    .option('fastly-serviceid', {
+      alias: ['fastlyNamespace', 'fastly-namespace'],
```

Comment:
also add `fastlyServiceid` as alias.

Reply:
Of course whoops! Got it

Reply:
I added the changes you requested! Good point, I will be sure to test changes more thoroughly next time; also thanks for the advice about sticking to camel case aliases!!!